scripts/publish-npm.js

```diff
@@ -0,0 +1,121 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const readline = require('readline');
+
+const rootPath = path.resolve('__dirname', '..');
+
+function requireSegmentApiKey() {
+  const MONGOSH_SEGMENT_API_KEY = process.env.MONGOSH_SEGMENT_API_KEY || '';
+
+  if (!MONGOSH_SEGMENT_API_KEY) {
+    throw new Error('MONGOSH_SEGMENT_API_KEY environment variable must be set');
```

Comment:
Since you have `assert` imported, I feel like this could be an `assert` statement as well.